### PR TITLE
Users/Admins/Assessors searching

### PR DESCRIPTION
**Pull request by jaleszek**

### CODE REVIEW

File: app/search/search.rb [hunk 1]
```diff
@@ -21,6 +21,7 @@ def initialize(scope)
     @ordered_by   = nil
     @ordered_desc = false
     @filter_params = {}
+    @query         = nil
```
Comment (matthewford) [inline app/search/search.rb]: Unnecessary spacing detected.

File: app/search/user_search.rb [hunk 2]
```diff
@@ -7,4 +7,5 @@ class UserSearch < Search
   }
 
   include FullNameSort
+
```
Comment (matthewford) [inline app/search/user_search.rb]: Extra empty line detected at class body end.

File: spec/features/admin/accounts/search_spec.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+include Warden::Test::Helpers
+
+describe "Users search", "
+  As Admin
+  I want to search users." do
+
```
Comment (matthewford) [inline spec/features/admin/accounts/search_spec.rb]: Extra empty line detected at block body beginning.

File: app/search/search.rb (hunk 1 above)
Comment (jaleszek) [inline app/search/search.rb]: probably can be ignored

File: spec/features/admin/accounts/search_spec.rb (hunk 3 above)
Comment (jaleszek) [inline spec/features/admin/accounts/search_spec.rb]: It doesn't look well without this line.

Comment (matthewford) [inline spec/features/admin/accounts/search_spec.rb]: some degree of human checking can be applied to the hound comments, not all will be applicable

**Pull request merged**